releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -0,0 +1,132 @@
+# STELLAR GMP v1.0.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @ahramy <ahram@interoplabs.io>       |
+| **Deployment** | @RiceAndMeet <steven@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Crates](https://crates.io/users/interoplabs-ci)
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases)
+
+## Background
+
+Changes in the release:
+
+1. This is the v1.0.0 initial GMP release.
+
+## Deployment
+
+Create an `.env` config.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=stellar
+```
+
+Deploy Stellar Gateway
+
+```bash
+# Deploy Gateway
+[1] node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
+```
+
+[2] After deploying the Stellar Gateway, ensure that you [deploy CosmWasm for Stellar GMP v1.0.0](../cosmwasm/2025-01-STELLAR-GMP-v1.0.0.md).
+
+- Once all steps are completed, update ampd with the Stellar chain configuration, then follow the remaining steps.
+
+```bash
+# Rotate genesis verifier set on Stellar Gateway
+[3] node stellar/gateway.js rotate --chain-name $CHAIN
+
+# Deploy Gas Service
+[4] node stellar/deploy-contract.js deploy axelar_gas_service --version v1.0.0
+
+# Deploy Operators
+[5] node stellar/deploy-contract.js deploy axelar_operators --version v1.0.0
+
+# Register refunder address as an operator
+[6] node stellar/operators.js --action add_operator -e devnet-amplifier --args $REFUNDER_ADDRESS
+
+# Deploy Example with dummy ITS address
+[7] node stellar/deploy-contract.js deploy example --version v1.0.0 --use-dummy-its-address
+
+```
+
+## Checklist
+
+The following checks should be performed after the rollout
+
+```bash
+ARGS=(
+  --keyring-backend test \
+  --from wallet \
+  --gas auto --gas-adjustment 1.4 \
+  --chain-id devnet-amplifier \
+  --node http://devnet-amplifier.axelar.dev:26657
+)
+```
+
+- [ ] Stellar → EVM GMP call
+
+```bash
+node stellar/gmp.js send avalanche 0x0FCb262571be50815627C16Eca1f5F3D342FF5a5 0x1234 CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC 1
+
+axelard tx wasm execute $GATEWAY '{"verify_messages":
```

Comment:
Remove these axelard cmds. Our Amplifier GMP docs can be referenced for these cmds